src/send/EnterAmount.tsx

```diff
@@ -441,14 +465,13 @@ function EnterAmount({
           />
         )}
 
-        {children}
```

Comment:
With children here, the bottom sheets were acting super weird. Need to go through and double check that this doesn't mess anything else up.